<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMedicine</class>
 <widget class="QDialog" name="AddMedicine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="overallLayout">
    <item>
     <widget class="QLabel" name="addMedsLabel">
      <property name="text">
       <string>Add Medicine</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addMedsInput"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttons">
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="avaliableMedsView"/>
    </item>
    <item>
     <widget class="QPushButton" name="removeMedsButton">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
